src/program.py

```diff
@@ -30,200 +30,164 @@
 
 class Program:
     def __init__(self, repo, name=None, installed=False):
-        '''
-        @param name (String) Name of the program
-        @param installed (bool) If the program is installed or not
-        '''
-        
-        ##Type checking params
+        ## Constructor
```

Comment:
Although I disagree on some minor points (adding a newline after the opening docstring), using tripple quotes (" or ' ) is quite standard in python

https://www.python.org/dev/peps/pep-0257/